source/main/gameplay/OutProtocol.cpp

```diff
@@ -149,65 +134,49 @@ bool OutProtocol::Update(float dt, Actor* truck)
     else if (truck && truck->ar_engine)
     {
         // truck and engine valid
-        if (truck->ar_engine->HasTurbo())
-        {
-            gd.Flags |= OG_TURBO;
-        }
-        gd.Gear = std::max(0, truck->ar_engine->GetGear() + 1); // we only support one reverse gear
-        gd.PLID = 0;
-        gd.Speed = fabs(truck->ar_wheel_speed);
-        gd.RPM = truck->ar_engine->GetEngineRpm();
-        gd.Turbo = truck->ar_engine->GetTurboPsi() * 0.0689475729f;
-        gd.EngTemp = 0; // TODO
-        gd.Fuel = 0; // TODO
+        if (truck->ar_engine->HasTurbo()) { gd.Flags |= OG_TURBO; }
+        gd.Gear        = std::max(0, truck->ar_engine->GetGear() + 1); // we only support one reverse gear
+        gd.PLID        = 0;
+        gd.Speed       = fabs(truck->ar_wheel_speed);
+        gd.RPM         = truck->ar_engine->GetEngineRpm();
+        gd.Turbo       = truck->ar_engine->GetTurboPsi() * 0.0689475729f;
+        gd.EngTemp     = 0; // TODO
+        gd.Fuel        = 0; // TODO
         gd.OilPressure = 0; // TODO
-        gd.OilTemp = 0; // TODO
+        gd.OilTemp     = 0; // TODO
 
         gd.DashLights = 0;
         gd.DashLights |= DL_HANDBRAKE;
         gd.DashLights |= DL_BATTERY;
         gd.DashLights |= DL_SIGNAL_L;
         gd.DashLights |= DL_SIGNAL_R;
         gd.DashLights |= DL_SIGNAL_ANY;
-        if (!truck->tc_nodash)
-            gd.DashLights |= DL_TC;
-        if (!truck->alb_nodash)
-            gd.DashLights |= DL_ABS;
+        if (!truck->tc_nodash) gd.DashLights |= DL_TC;
+        if (!truck->alb_nodash) gd.DashLights |= DL_ABS;
 
         gd.ShowLights = 0;
-        if (truck->ar_parking_brake)
-            gd.ShowLights |= DL_HANDBRAKE;
-        if (truck->ar_lights)
-            gd.ShowLights |= DL_FULLBEAM;
-        if (truck->ar_engine->HasStarterContact() && !truck->ar_engine->IsRunning())
-            gd.ShowLights |= DL_BATTERY;
-        if (truck->ar_left_blink_on)
-            gd.ShowLights |= DL_SIGNAL_L;
-        if (truck->ar_right_blink_on)
-            gd.ShowLights |= DL_SIGNAL_R;
-        if (truck->ar_warn_blink_on)
-            gd.ShowLights |= DL_SIGNAL_ANY;
-        if (truck->tc_mode)
-            gd.ShowLights |= DL_TC;
-        if (truck->alb_mode)
-            gd.ShowLights |= DL_ABS;
+        if (truck->ar_parking_brake) gd.ShowLights |= DL_HANDBRAKE;
+        if (truck->ar_lights) gd.ShowLights |= DL_FULLBEAM;
+        if (truck->ar_engine->HasStarterContact() && !truck->ar_engine->IsRunning()) gd.ShowLights |= DL_BATTERY;
+        if (truck->ar_left_blink_on) gd.ShowLights |= DL_SIGNAL_L;
+        if (truck->ar_right_blink_on) gd.ShowLights |= DL_SIGNAL_R;
+        if (truck->ar_warn_blink_on) gd.ShowLights |= DL_SIGNAL_ANY;
+        if (truck->tc_mode) gd.ShowLights |= DL_TC;
+        if (truck->alb_mode) gd.ShowLights |= DL_ABS;

```

Comment:
here, I'd like it column-indented:
```
if (truck->tc_mode)     { gd.ShowLights |= DL_TC; }
if (truck->alb_mode)    { gd.ShowLights |= DL_ABS; }
```
but then it'd be ruined by the `HasStarterContact..IsRunning` long condition, so maybe it's best this way.